torch/_inductor/fx_passes/overlap_scheduling.py

```diff
@@ -289,6 +289,7 @@ def __init__(
         collective_estimator: Literal["analytical", "benchmark"],
         max_memory_increase_gb: float | None = 1.0,
         max_memory_increase_ratio: float | None = 0.05,
+        log_final_collectives_estimations: bool = False,
```

Comment:
since this doesnt affect functionality, just logging, maybe we remove it as a top level arg and just config it somewhere.

Reply:
Yeah, I worried about the cases when OverlapScheduler will be forked and global deps will need some afterwork and decided to avoid them.